packages/airnode-utilities/src/gas-prices/types.ts

```diff
@@ -0,0 +1,18 @@
+import { BigNumber } from 'ethers';
+
+export interface GasTarget {
+  readonly maxPriorityFeePerGas?: BigNumber;
```

Comment:
I'm not sure about all of these `readonly` modifiers in a "utilities" library. There might be use cases where the implementer needs mutation and this makes it quite restrictive. I know @Siegrift isn't a fan of `readonly` 😁

Reply:
I removed `readonly` from the gas-prices types, but it looks like the other utility modules also have them :thinking:

Reply:
I dislike it because sometimes I want to write mutable code and this readonly modifier is in the way. This is especially true in tests, where you want to sometimes override a certain property. And lastly, it feels too verbose to me.